src/Command/RefreshCommand.php

```diff
@@ -51,54 +99,54 @@ public function configure()
         $this->addOption('session', null, InputOption::VALUE_OPTIONAL, 'The session to use for this command');
     }
 
+    protected function interact(InputInterface $input, OutputInterface $output)
+    {
+        parent::interact($input, $output);
+
+        // TODO: check if it is phpcr check for session or if it is orm do not accept session
+    }
+
     /**
      * {@inheritdoc}
      */
     public function execute(InputInterface $input, OutputInterface $output)
     {
-        $container = $this->getContainer();
-        $manager = $container->get('doctrine_phpcr');
-        $factory = $container->get('cmf_routing_auto.metadata.factory');
-        $arm = $container->get('cmf_routing_auto.auto_route_manager');
-
-        $dm = $manager->getManager();
-        $uow = $dm->getUnitOfWork();
-
         $session = $input->getOption('session');
         $dryRun = $input->getOption('dry-run');
         $class = $input->getOption('class');
         $verbose = $input->getOption('verbose');
 
-        DoctrineCommandHelper::setApplicationPHPCRSession(
-            $this->getApplication(),
-            $session
-        );
+        if ('doctrine_phpcr_odm' === $this->adapterName) {
+            DoctrineCommandHelper::setApplicationPHPCRSession(
+                $this->getApplication(),
+                $session
+            );
+        }
```

Comment:
Code smell. This could _maybe_ be part of a PHPCR-ODM refresh routes service as mentioned below. I actually forget why we need this at all, seems like a legacy mistake.